src/main/java/com/sngular/kloadgen/config/schemaregistry/SchemaRegistryConfigElement.java

```diff
@@ -78,6 +86,14 @@ private Map<String, String> getProperties() {
     return result;
   }
 
+  private String getRegistryName() {
+    String registryName = getPropertyAsString("schemaRegistryName");
```

Comment:
I think it would be a nice idea to create an static variable with this value like you have done in SchemRegistryConfigElementBeanInfo.java instead of hardcoding the value.